torch/utils/_config_module.py

```diff
@@ -13,6 +13,27 @@
 from typing_extensions import deprecated
 from unittest import mock
 
+from torch._utils_internal import justknobs_check
+
+
+@dataclass
+class Config:
+    """Represents a config with richer behaviour than just a default value.
+
+    i.e.
+    foo = Config(justknob="//foo:bar", default=False)
+    install_config_module(...)
```

Comment:
If you want this to format correctly in restructured text, you should do something like:

```
::
    foo = Config(justknob="//foo:bar", default=False)
    install_config_module(...)
```

Reply:
done.